lib/import-export-visitor.js

```diff
@@ -211,21 +242,37 @@ module.exports = class ImportExportVisitor extends Visitor {
 
         if (this.modifyAST) {
           let ast = this.parse(codeToInsert);
+
           if (ast.type === "File") ast = ast.program;
           assert.strictEqual(ast.type, "Program");
+
           const spliceArgs = ast.body;
           spliceArgs.unshift(bodyInfo.insertNodeIndex, 0);
+
           const body = bodyInfo.body;
           body.splice.apply(body, spliceArgs);
+
+          const parsedDirectives = ast.directives;
+          const parentDirectives = bodyInfo.parent.directives;
+
+          if (parsedDirectives &&
+              parentDirectives) {
+            parentDirectives.push.apply(
+              parentDirectives,
+              parsedDirectives
+            );
```

Comment:
Does this only apply for Babylon since it requires the `.directives` property. How are non-Babylon handled?